services/movie.py

```diff
@@ -0,0 +1,45 @@
+import init_django_orm  # noqa: F401
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None,
+               actors_ids: list = None) -> Movie:
```

Comment:
You should specify inner type of list
```suggestion
def get_movies(genres_ids: list[int] = None,
               actors_ids: list[int] = None) -> Movie:
```